src/index.js

```diff
@@ -132,6 +145,7 @@ async function scaffold(to, opts) {
 
 		const htmlPath = resolve(to, 'index.html');
 		const html = (await fs.readFile(htmlPath, 'utf-8')).replace('index.jsx', 'index.tsx');
+		// TODO: should we also rename all current jsx files to tsx?
```

Comment:
Yeah, I think I can move this to the template copy instead, and should, especially now that TS supports using `.ts` in import specifiers. Bad hack on my part, sorry.